transforms-caching-experiment/remote-cache-experiment-transforms.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+
+# Note:
```

Comment:
I think you could just merge the information here into the README. Otherwise, it feels like the documentation is spread across two places.

Reply:
Done